pallets/utility/src/lib.rs

```diff
@@ -180,7 +180,7 @@ decl_module! {
         /// event is deposited.
         #[weight = <T as Config>::WeightInfo::batch(&calls)]
         pub fn batch(origin, calls: Vec<<T as Config>::Call>) {
-            let is_root = Self::ensure_root_or_signed(origin.clone()).is_ok();
+            let is_root = Self::ensure_root_or_signed(origin.clone())?;
```

Comment:
Documentation above says "will return `Ok` in all circumstances

Reply:
It will only error if the transaction is not signed.

Reply:
I think the whole batch call should be rejected if `origin` is not signed.

Reply:
Can you update the docs to reflect this change?

Reply:
And some changelogs as well

Reply:
The docs are the same on all the `batch*` extrinsics.

Reply:
Then the docs are wrong on all:
> May be called from any origin.

But `ensure_none` is a valid origin not accepted.